mst/billing.md

```diff
@@ -41,12 +41,91 @@ Migrating a service to another cloud region or different cloud provider does not
 incur extra charges.
 
 <Highlight type="note">
-All prices listed for Managed Service for TimescaleDB are inclusive of credit
-card and processing fees. However, in some cases, your credit card provider
-might charge additional fees, such as an international transaction fee. These
-fees are not charged by Timescale or Aiven.
+All prices listed for Managed Service for TimescaleDB are inclusive of 
+credit card and processing fees. However, in some cases, your credit card 
+provider might charge additional fees, such as an international transaction 
+fee. These fees are not charged by Timescale or Aiven.
 </Highlight>
 
+## Billing groups
+
+Create billing groups to set up common billing profiles for projects within an 
+organization. Billing groups make it easier to manage your costs since you 
+receive a consolidated invoice for all projects assigned to a billing group 
+and can pay with one saved payment method. 
+
+Billing groups can only be used in one organization. Aiven 
+credits are assigned per billing group and are automatically used to 
+cover charges of any project assigned to that group.
+
+You can track spending by exporting cost information to business 
+intelligence tools using the [invoice API][invoice-api].
+
+To access billing groups in the [Aiven Console][aiven-console], you must be a
```

Comment:
Careful here, you are using MST not Aiven:

https://portal.managed.timescale.com/account/a4bf28029b59/billing/overview

Billing groups are in:

https://portal.managed.timescale.com/account/a4bf28029b59/billing/billing-groups